app/src/test/java/com/x8bit/bitwarden/data/platform/datasource/disk/EnvironmentDiskSourceTest.kt

```diff
@@ -25,11 +28,9 @@ class EnvironmentDiskSourceTest {
 
     @Test
     fun `preAuthEnvironmentUrlData should pull from and update SharedPreferences`() {
-        val environmentKey = "bwPreferencesStorage:preAuthEnvironmentUrls"
-
         // Shared preferences and the repository start with the same value.
         assertNull(environmentDiskSource.preAuthEnvironmentUrlData)
-        assertNull(fakeSharedPreferences.getString(environmentKey, null))
+        assertNull(fakeSharedPreferences.getString(PRE_AUTH_URLS_KEY, null))
```

Comment:
Why change this?

Reply:
Just for consistency and future reusability.